test/identify-test.js

```diff
@@ -129,16 +141,26 @@ describe('API Blueprint', function() {
     });
   });
 
-  describe('without FORMAT header or any typical response', function() {
+  describe('without FORMAT header or any typical response but with Data Structures', function() {
     const source = dedent`
       # Data Structures API
 
-      ## Data Structures
+      ## Data Structure
```

Comment:
Why here it is `Data Structure` and in the test title it is `Data Structures`? The spec says it should be `Data Structures`: https://apiblueprint.org/documentation/specification.html#def-data-structures If this passes, then it's false positive test.

Reply:
Oh, I see there is `structures?`. That's not clear from neither the spec or the test description. It should be actually two tests, IMHO.

Reply:
Yeah I agree, we should definately have a test for the form that we tell users in our documentation.